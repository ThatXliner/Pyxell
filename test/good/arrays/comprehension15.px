
a: [Float] = [x*10^^15 for x in 2..4]
print a
