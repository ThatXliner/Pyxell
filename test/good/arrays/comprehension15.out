[2e+15, 3e+15, 4e+15]
